### PR TITLE
Remove duplicate log lines from sbt-server

### PR DESCRIPTION
**Pull request by jsuereth**
This removes duplicated log lines by ignoring lines that are sent to a semantic logger AND println.

### CODE REVIEW

**Comment by jsuereth** [comment]
Review by @corruptmemory

File: server/src/main/scala/sbt/server/ServerLogging.scala [hunk 1]
```diff
@@ -26,7 +26,12 @@ private[sbt] abstract class EventLogger[E <: protocol.LogEvent](protected val lo
   }
 
   def log(level: Level.Value, message: => String): Unit = {
-    send(protocol.LogMessage(level.toString, message))
+    // Here we're trying to catch when the message already has the log level in it, for some reason.
+    val m = message
+    val msg = logLevelRegex.findFirstMatchIn(m).flatMap { m =>
+      Some(m.group("message"))
+    } getOrElse (m)
```
Comment (corruptmemory) [inline server/src/main/scala/sbt/server/ServerLogging.scala]: HA! A few too many references to an identifier `m`.  It's all good, but perhaps a little more creativity on the identifier selection. :-)

Comment (jsuereth) [inline server/src/main/scala/sbt/server/ServerLogging.scala]: To be fair, I think this code is no longer used :)...

**Comment by corruptmemory** [comment]
LGTM

**Pull request merged**